src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,12 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    /**
+     *
+     * @return true if data is mutated by command
+     */
```

Comment:
The start of the header comment does not follow the convention (i.e. first sentence should be something like `Selects next line …` instead of `Select next line` or `This method is for selecting …`)

It is good to add header comment so users of the method as well as those who overrides method will know how the method behaves =) , do follow the commenting style as per our coding standard!